app/src/components/common/layout/header/index.tsx

```diff
@@ -41,13 +47,13 @@ export const HeaderInner = styled.div`
   justify-content: space-between;
   margin: 0 auto;
   max-width: 100%;
-  padding: 0 10px;
+  padding: 24px 0px;
   position: relative;
   width: ${props => props.theme.themeBreakPoints.xxl};
 
-  @media (min-width: ${props => props.theme.themeBreakPoints.md}) {
-    padding: 0 ${props => props.theme.paddings.mainPadding};
-  }
+  // @media (min-width: ${props => props.theme.themeBreakPoints.md}) {
+  //   padding: 30px 10px;
+  // }
```

Comment:
this as well...well probably do refactor of those in next release so we should keep it clean!

Reply:
yes sure I did too